### ISSUE TITLE
WORLD_PRESET="DST_CAVE" does not work

### ISSUE THREAD

**Issue by averri** [yeet-zone/docker-dontstarvetogether#108]
Using the environment variable WORLD_PRESET="DST_CAVE" does not take any effect. The Caves is not enabled in the server.

**Comment by thasmo** [comment]
I've invested quite some time now to figure out what causes this and the problem seems to be our distributed `leveldataoverride.lua` configuration, which is missing some cave-related settings so the server is able to start a cave-server. In fact, to be able to run a cave-server several settings need to be defined. This has changed with the introduction of the `leveldataoverride.lua` file and it seems this got more complicated than it was before. I'm tempted to removed the feature, which we implemented to start a cave-server fast and just support setting the `leveldataoverride.lua` file's contents properly.

In other words; at the moment this doesn't work properly but you can set the `LEVELDATA_OVERRIDES` env-variable to populate the `leveldataoverride.lua` file, which can be configured to run a custom cave server.

**Comment by BraisGabin** [comment]
This is a bug but it seems an "I wont fix". Klei did this too complicated. We are talking about alternatives in #109.

**Comment by thasmo** [comment]
Closing this in favor of #109. @averri, please feel free to ask if you need help using the `LEVELDATA_OVERRIDES` option.
